remix.init/remix.config.js

```diff
@@ -1,10 +1,14 @@
 const { config } = require("@netlify/remix-edge-adapter");
+const baseConfig =
+  process.env.NODE_ENV === "production"
```

Comment:
Only use the Netlify Edge function to serve pages if we are buidling for production. Otherwise, this handles all requests when running `netlify dev` where we want the Remix dev server running under the hood.